svelte/components/Collection/CollectionCreate.svelte

```diff
@@ -230,8 +229,8 @@
           </div>
 
           {#if template === "OpenAutoMarket/ownable"}
-            <div class="section">
```

Comment:
should be display if price > 0 and not collection owner

Reply:
collection is not created so price and owner are undefined ?

Reply:
ok, so to display with no condition ?